test failing

@claude Run npm test
npm error Missing script: "test"
npm error
npm error To see a list of scripts, run:
npm error   npm run
npm error A complete log of this run can be found in: /home/runner/.npm/_logs/2025-12-28T13_23_10_592Z-debug-0.log
Error: Process completed with exit code 1.

--- a/embuild-analyses/package.json
+++ b/embuild-analyses/package.json
@@ -7,7 +7,7 @@
     "build": "npm run validate:embeds && next build",
     "start": "next start",
     "lint": "eslint",
-    "test": "npm run lint && npm run build",
+    "test": "playwright test",
     "validate:embeds": "node scripts/validate-embed-paths.js",
     "check:faillissementen-geo-join": "node scripts/check-faillissementen-geo-join.js"
   },

